### PR TITLE
Users/jysin/manual validation task

### PR DESCRIPTION
**Pull request by Josh-01**
**Task name**: Manual Validation

**Description**: task.json changes for the server task

**Documentation changes required:** (Y/N) Y
**Added unit tests:** (Y/N) No

**Attached related issue:** (Y/N) N

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/ManualValidationV0/task.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            "name": "onTimeout",
+            "type": "radio",
+            "label": "On timeout",
+            "defaultValue": "reject",
+            "required": false,
+            "options": {
+                "reject": "Reject",
+                "resume": "Resume"
+            },
+            "helpMarkDown": "Reject or resume this manual intervention automatically after it is pending for the specified timeout or 60 days, whichever is earlier."
```
Comment (rohit-batra) [inline Tasks/ManualValidationV0/task.json:53]: Can we please confirm if its 60 days (Also can we add the default option here as in what will be the default behavior as this input is non-mandatory)

File: Tasks/ManualValidationV0/task.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        "Patch": 1
+    },
+    "inputs": [
+        {
+            "name": "validators",
+            "type": "identities",
+            "label": "Manual Validators",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Only users added in validators list can act on a manual validation."
```
Comment (rohit-batra) [inline Tasks/ManualValidationV0/task.json:28]: We need to be sure of this also . In MI , there is a separate Security bit which gives ability to be able to Intervene , and this is for notification purpose only. Can we confirm once just .

File: Tasks/ManualValidationV0/task.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                "resizable": "true",
+                "rows": "10",
+                "maxLength": "4000"
+            }
+        },
+        {
+            "name": "onTimeout",
+            "type": "radio",
+            "label": "On timeout",
+            "defaultValue": "reject",
```
Comment (rohit-batra) [inline Tasks/ManualValidationV0/task.json:47]: Default was to be resume as per Raiyan, right

File: Tasks/ManualValidationV0/task.json (hunk 1 above)
Comment (rohit-batra) [inline Tasks/ManualValidationV0/task.json:53]: Can we please confirm if its 60 days (Also can we add the default option here as in what will be the default behavior as this input is non-mandatory)

Comment (Josh-01) [reply Tasks/ManualValidationV0/task.json]: It was 30 days, so changed that. If it is not specified, default is taken as 'reject'

File: Tasks/ManualValidationV0/task.json (hunk 3 above)
Comment (Josh-01) [reply Tasks/ManualValidationV0/task.json:47]: Default is reject (confirmed)

File: Tasks/ManualValidationV0/task.json (hunk 2 above)
Comment (Josh-01) [reply Tasks/ManualValidationV0/task.json]: This field is only to specify who we send emails to. I have corrected the description

**Pull request merged**